hack/scripts/run-non-olm-kuttl-test.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+export NON_OLM="true"
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+sequential_suite=$ROOT/../../test/openshift/e2e/sequential/
+parallel_suite=$ROOT/../../test/openshift/e2e/parallel/
+
+testsuite="sequential"
+
+# Get test suite argument
+while getopts ":t:" opt; do
+  case ${opt} in
+    t) testsuite=$OPTARG;;
+    \?) echo "Please provide options sequential or parallel -$OPTARG" >&2;;
+  esac
+done
+
+# these tests will be removed while running non-olm operator test
+# 1-031_validate_toolchain
+# 1-085_validate_dynamic_plugin_installation
+# 1-036_validate_keycloak_resource_reqs
+# 1-038_validate_productized_images
+# 1-051-validate_csv_permissions
+# 1-073_validate_rhsso
+# 1-077_validate_disable_dex_removed
+# 1-090_validate_permissions
+
+if [ -n "$EXCLUDED_TESTS" ]; then
+  filenames=$EXCLUDED_TESTS
+else 
+  filenames="1-031_validate_toolchain 1-085_validate_dynamic_plugin_installation 1-036_validate_keycloak_resource_reqs 1-038_validate_productized_images 1-051-validate_csv_permissions 1-073_validate_rhsso 1-077_validate_disable_dex_removed 1-090_validate_permissions"
+fi
+
+for dir in $filenames ; do
+  if [ -d "$sequential_suite/$dir" ]; then
+    echo "Deleting directory $dir"
+    rm -rf "$sequential_suite/$dir"
```

Comment:
Is there a plan to restore these deleted test directories? After the execution, we are left with tests that are only specific to non-OLM way installed operator. Can we have an alternative for this? Maybe create a temporary directory to place these and copy them back after the execution or something similar?

Reply:
Good idea, let me add something for it.